hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/TestContainerCache.java

```diff
@@ -85,17 +86,17 @@ public void testContainerCacheEviction() throws Exception {
 
     // Get 2 references out of the same db and verify the objects are same.
     ReferenceCountedDB db1 = cache.getDB(1, "RocksDB",
-        containerDir1.getPath(), conf);
+            containerDir1.getPath(), OzoneConsts.SCHEMA_LATEST, conf);
```

Comment:
This could be done, but it may not be clear to the caller that the latest schema version would be applied. Since there is no mention of it in the method name, the caller may assume that the schema version would be automatically determined based on some property of the DB. Aside from this series of calls in one test, the method is only called once in the code, so I think it is OK to leave it the way it is for clarity.